src/seismometer/_api.py

```diff
@@ -130,7 +130,8 @@ def target_feature_summary(exclude_cols: list[str] = None, inline=False):
     """
     sg = Seismogram()
 
-    exclude_cols = exclude_cols or sg.entity_keys
+    exclude_cols = exclude_cols or [key for key in sg.entity_keys]
+    exclude_cols += [pdh.event_time(event.display_name) for event in sg.config.events.values()]
```

Comment:
I think using [select_dtypes](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.select_dtypes.html) will be more robust to usecase-specific columns and changes to event class

Reply:
done

Reply:
had to include "datetime64[ns]" as well